### DISCUSSION TITLE
electron 应用。使用prisma作为本地数据库。

### DISCUSSION THREAD

**Discussion by guagezuishuai** [prisma/prisma#23291] [Q&A]
### Question

electron构建的一个桌面应用，使用prisma作为本地数据库。当需要数据库变更或者数据迁移的时候我应该怎么做？

用户电脑没有对应环境，不能跑已经提供的指令。

**Translation in English**
A desktop application built by electron, using prisma as the local database. What should I do when database changes or data migration are required?

The user's computer does not have a corresponding environment and cannot run the provided instructions.

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @guagezuishuai 👋 

You can use [Prisma Migrate](https://www.prisma.io/docs/orm/prisma-migrate) for managing data migration within your Electron application. There is an open issue that shows how other users use Prisma within their Electron App..

- https://github.com/prisma/prisma/issues/9613

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
